bitnami/redmine/values.yaml

```diff
@@ -396,18 +422,11 @@ ingress:
   ## @param ingress.enabled Enable ingress record generation for Redmine
   ##
   enabled: false
-  ## @param ingress.className Set the ingressClassName which will be used for this ingress
-  ## The newer ingressClassName field on Ingresses is a replacement for that "kubernetes.io/ingress.class" annotation (Kubernetes 1.18+).
-  ## When using this filed you need to create the IngressClass resource with the corresponding name.
+  ## @param ingress.className IngressClass that will be be used to implement the Ingress (Kubernetes 1.18+)
+  ## This is supported in Kubernetes 1.18+ and required if you have more than one IngressClass marked as the default for your cluster .
+  ## ref: https://kubernetes.io/blog/2020/04/02/improvements-to-the-ingress-api-in-kubernetes-1.18/
   ##
   className: ""
```

Comment:
Could you please add support for `ingressClassName` using `coalesce`? Don't want to bump a major, but would like the charts to at least support the same values.